ci/dev.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:12
```

Comment:
Let's put this in `ci/dev-image` along with the script to start it up and name that `start.sh` instead.

Reply:
Actually, how about `./exec.sh`?

Reply:
VSCode needs the `.Dockerfile` for language features.... it is not the same in other editors?

Reply:
VSCode needs the `.Dockerfile` for language features.... it is not the same in other editors?

Reply:
ci/dev-image would be a directory with Dockerfile and exec.sh

Reply:
oh I see. sounds good 👍